lua/telescope/previewers/buffer_previewer.lua

```diff
@@ -376,13 +503,17 @@ previewers.vimgrep = defaulter(function(opts)
 
       conf.buffer_previewer_maker(p, self.state.bufnr, {
         bufname = self.state.bufname,
+        winid = self.state.winid,
+        preview_timeout = opts.preview_timeout,
+        preview_timeout_hook = opts.preview_timeout_hook,
+        preview_filesize_limit = opts.preview_filesize_limit,
         callback = function(bufnr)
           jump_to_line(self, bufnr, entry.lnum)
         end,
       })
     end,
   }
-end, {})
```

Comment:
You changed `defaulter` and removed `, {})` for the two important onces. But dont you have to adjust this for all other previewers as well?

Reply:
Ah sorry, I first removed them and then thought maybe there are cases I'm not seeing and wanted to revert all of them to as it was befote. Thought I did, I'll check once more.

Reply:
Ah sorry, I first removed them and then thought maybe there are cases I'm not seeing and wanted to revert all of them to as it was befote. Thought I did, I'll check once more.

Reply:
I think I got them all reverted.